File: assignemnt soln/MainActivity.java
package com.example.fragmentobjectpass;

import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


	fragment1 fragment1=new fragment1();
        FragmentManager fragmentManager=getSupportFragmentManager();
        FragmentTransaction fragmentTransaction=fragmentManager.beginTransaction();
        
        fragmentTransaction.replace(R.id.frmid,fragment1).commit();
    }
}

File: Module 2/26-12/MainActivity.kt
package com.example.layoutex1

import android.os.Bundle
import android.widget.LinearLayout
import android.widget.TextView
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

class MainActivity : AppCompatActivity()
{
    lateinit var linearLayout: LinearLayout
    lateinit var txt1:TextView

    override fun onCreate(savedInstanceState: Bundle?)
    {
        super.onCreate(savedInstanceState)
       // setContentView(R.layout.activity_main)
        linearLayout = LinearLayout(this)
        txt1 = TextView(this)
        txt1.setText("Hello")

        var width = 500
        var height = 500

        txt1.width=width
        txt1.height=height

        linearLayout.addView(txt1,width,height)
        setContentView(linearLayout)



    }
}